src/components/group/GroupPickerDialog.tsx

```diff
@@ -100,9 +100,10 @@ export const GroupPickerDialog = ({
   );
 
   const isRowDisabled = (row?: GroupRepresentation) => {
-    return !![...joinedGroups, ...(filterGroups || [])].find(
-      (group) => group.id === row?.id
-    );
+    return !![
+      ...joinedGroups.map((item) => item.name),
+      ...(filterGroups || []),
+    ].find((group) => group === row?.name);
```

Comment:
This code can be cleaned up a bit by using [`.some()` ](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)instead of `.find()`.

```suggestion
    return [
      ...joinedGroups.map((item) => item.name),
      ...(filterGroups || []),
    ].some((group) => group === row?.name);
```

Reply:
Ah nice catch! It definitely makes more sense to use `.some()` here